src/integrationTests/secureSignal.test.ts

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2021 The Trade Desk, Inc
```

Comment:
You can remove the copyright notice - we just keep the one version for the whole repo now.